app/src/main/res/values/strings.xml

```diff
@@ -643,6 +644,7 @@
     <string name="editactionfeed_translate_title_dialog_learn_more">Learn more</string>
     <string name="editactionfeed_to_language">To: </string>
     <string name="editactionfeed_added_by_you">Added by you:</string>
+    <string name="editactionfeed_translated_by_you">Translated by you:</string>
```

Comment:
The title of the card screen should be changed to `Translate descriptions` (remove the **title**)

Reply:
I have changed both the title string and DescriptionEditActivity title both to 'Translate descriptions' . I will change the DescriptionEditActivity title to 'Translate description' if Robin feels it has to be singular.